components/action-mgt/org.wso2.carbon.identity.action.execution/pom.xml

```
@@ -100,7 +104,9 @@
                             org.apache.http.impl.conn; version="${httpcomponents-httpclient.imp.pkg.version.range}",
                             org.osgi.framework; version="${osgi.framework.imp.pkg.version.range}",
                             org.osgi.service.component; version="${osgi.service.component.imp.pkg.version.range}",
+                            org.wso2.carbon.identity.central.log.mgt.utils; version="${carbon.identity.package.import.version.range}",
                             org.wso2.carbon.identity.core.util; version="${carbon.identity.package.import.version.range}",
+                            org.wso2.carbon.identity.core.*; version="${carbon.identity.package.import.version.range}",
```

```suggestion
                            org.wso2.carbon.identity.core; version="${carbon.identity.package.import.version.range}",
```
Hope we don't need to import all the packages inside `org.wso2.carbon.identity.core`